recipes/fossil/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "fossil" %}
+{% set version = "2.7" %}
+{% set am_version = "1.15" %} # keep synchronized with build.sh
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://www.fossil-scm.org/xfer/uv/{{ name }}-src-{{ version }}.tar.gz
+  sha256: 2c24359131d7495e47dc95021eb35f1ba408ded9087e36370d94742a4011033c
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - ccache  # [unix]
+    - cmake  # [win]
+    - make  # [unix]
+    - perl  # [win]
+  host:
+    - openssl
+    - sqlite
+    - tk
+    - zlib
+  run:
+    - openssl
+    - sqlite
+    - tk
+    - zlib
+
+test:
+  commands:
+    - fossil --help
+
+about:
+  home: https://fossil-scm.org
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: COPYRIGHT-BSD2.txt
+  summary: >
+    Fossil is a simple, high-reliability, distributed software configuration
+    management system with these advanced features:
+
+  doc_url: https://fossil-scm.org/index.html/doc/trunk/www/permutedindex.html
+  dev_url: https://github.com/simplejson/simplejson
```

Comment:
Just noticed this URL looked off. Have raised as issue ( https://github.com/conda-forge/fossil-feedstock/issues/4 ).